src/sentry/interfaces/base.py

```diff
@@ -67,7 +67,7 @@ def __init__(self, **data):
         self._data = data or {}
 
     def __eq__(self, other):
-        if type(self) != type(other):
+        if not isinstance(self, type(other)):
```

Comment:
can we change this to `if type(self) is not type(other)`? Isinstance checks should not be done here.

Reply:
The linter automatically changes this for me :(

Reply:
No dice. The linter overrules this.